Magic/src/main/java/com/elmakers/mine/bukkit/action/builtin/ChatAction.java

```diff
@@ -0,0 +1,73 @@
+package com.elmakers.mine.bukkit.action.builtin;
+
+import com.elmakers.mine.bukkit.action.BaseSpellAction;
```

Comment:
Correct me if I'm wrong, but this import order doesn't look correct.
Does this pass checkstlye? (/ A maven build)

Reply:
Make sure to try running maven from the command line and see if it succeeds.
We have configured a few maven plugins for consistent styling.

Import order goes
- java(x)
- org
- others

IIRC. You can check out other files as well.